2019/5xxx/CVE-2019-5624.json

```diff
@@ -1,18 +1,127 @@
+    "ASSIGNER": "cve@rapid7.com",
+    "DATE_PUBLIC": "2019-04-24T18:00:00.000Z",
+    "TITLE": "Rapid7 Metasploit Framework Zip Import Directory Traversal",
+    "AKA": "",
+    "STATE": "PUBLIC"
+  },
+  "source": {
+    "defect": [
+      "MS-3923"
+    ],
+    "advisory": "",
+    "discovery": "UNKNOWN"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "Rapid7",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Metasploit Framework",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_name": "4.14.0",
+                      "version_affected": "<=",
+                      "version_value": "4.14.0",
+                      "platform": ""
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "problemtype": {
+    "problemtype_data": [
+      {
+        "description": [
+          {
+            "lang": "eng",
+            "value": "CWE-22"
+          },
+          {
+            "lang": "eng",
+            "value": "Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal')"
+          }
         ]
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "Rapid7 Metasploit Framework suffers from an instance of CWE-22, Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal') in the Zip import function of Metasploit. Exploiting this vulnerability can allow an attacker to execute arbitrary code in Metasploit at the privilege level of the user running Metasploit."
+      },
+      {
+        "lang": "eng",
+        "value": "This issue affects:\nRapid7 Metasploit Framework version 4.14.0 and prior versions."
+      }
+    ]
+  },
+  "references": {
+    "reference_data": [
+      {
+        "refsource": "CONFIRM",
+        "url": "https://github.com/rapid7/metasploit-framework/pull/11716",
+        "name": "https://github.com/rapid7/metasploit-framework/pull/11716"
+      },
+      {
+        "refsource": "CONFIRM",
+        "url": "https://help.rapid7.com/metasploit/release-notes/archive/2019/04/#20190416",
+        "name": "https://help.rapid7.com/metasploit/release-notes/archive/2019/04/#20190416"
+      },
+      {
+        "refsource": "MISC",
+        "url": "https://blog.doyensec.com/2019/04/24/rubyzip-bug.html",
+        "name": "https://blog.doyensec.com/2019/04/24/rubyzip-bug.html"
+      }
+    ]
+  },
+  "configuration": [],
+  "impact": {
+    "cvss": {
+      "version": "3.0",
+      "attackVector": "LOCAL",
+      "attackComplexity": "LOW",
+      "privilegesRequired": "HIGH",
+      "userInteraction": "REQUIRED",
+      "scope": "CHANGED",
+      "confidentialityImpact": "HIGH",
+      "integrityImpact": "HIGH",
+      "availabilityImpact": "NONE",
```

Comment:
If they can run arbitrary code as Metasploit service instance user couldn't they take the instance down?

Reply:
Yes! But generally speaking, that's a post-exploit scenario -- exercising this vuln (in isolation) gets you C:H and I:H, but to kill the system/service, you'd need to Do A Thing specifically for that. Anyway, I think most people count "Availability" as a measurement of a DoS attack.

And you can file this all under the "Why CVSSv3 isn't Great" header.

Reply:
That's a helpful separation on Availability, thanks!